retroshare-gui/src/gui/statistics/GlobalRouterStatistics.cpp

```diff
@@ -177,6 +180,18 @@ void GlobalRouterStatistics::updateContent()
         
         if(nicknames.isEmpty())
           nicknames = tr("Unknown");
+	  
```

Comment:
You should directly use a GxsIdTreeWidgetItem: it manages the display of avatar, a proper tooltip, etc!